### PR TITLE
Review platform api collection responses

### PR DESCRIPTION
**Pull request by ironBatista**
#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

#### Description
This issue ensures that API response collections match columns from Klear lists under the Platform section.

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

File: library/Ivoz/Provider/Domain/Model/Domain/DomainDto.php [hunk 1]
```diff
@@ -15,6 +15,7 @@ public static function getPropertyMap(string $context = '', string $role = null)
                 'id' => 'id',
                 'domain' => 'domain',
                 'pointsTo' => 'pointsTo'
+                // @TODO client
```
Comment (mmadariaga) [inline library/Ivoz/Provider/Domain/Model/Domain/DomainDto.php:18]: Shouldn't this be part of this PR as well?

**Review by mmadariaga — CHANGES_REQUESTED** [review]

**Comment by ironArt3mis** [comment]
Rebase completed.

File: library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/BrandDoctrineRepository.php [hunk 2]
```diff
@@ -41,4 +42,20 @@ public function getNames()
...
     }
+
+    public function findOneByDomain(string $domainUsers): ?BrandInterface
+    {
+        $qb = $this->createQueryBuilder('self');
+
+        $qb
+            ->select('self')
+            ->setParameter(':domainUsers', $domainUsers)
+            ->where('self.domainUsers in (:domainUsers)');
```
Comment (mmadariaga) [inline library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/BrandDoctrineRepository.php:53]: in?

File: library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/CompanyDoctrineRepository.php [hunk 3]
```diff
@@ -181,4 +181,20 @@ private function getIdsByBrandAndType(int $brandId, string $type): array
...
     }
+
+    public function findOneByDomain(string $domainUsers): ?CompanyInterface
+    {
+        $qb = $this->createQueryBuilder('self');
+
+        $qb
+            ->select('self')
+            ->setParameter(':domainUsers', $domainUsers)
+            ->where('self.domainUsers in (:domainUsers)');
```
Comment (mmadariaga) [inline library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/CompanyDoctrineRepository.php:192]: in?

File: web/rest/platform/features/provider/notificationTemplate/postNotificationTemplate.feature [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    """
+
+  Scenario: Retrieve created notification template
+    Given I add Authorization header
+    When I add "Accept" header equal to "application/json"
+    And I send a "GET" request to "notification_templates/5"
+    Then the response status code should be 200
+    And the response should be in JSON
+    And the header "Content-Type" should be equal to "application/json; charset=utf-8"
+    And the JSON should be like:
```
Comment (mmadariaga) [inline web/rest/platform/features/provider/notificationTemplate/postNotificationTemplate.feature:38]: No reason to use "like" here. Use `the JSON should be equal to` instead

**Review by mmadariaga — CHANGES_REQUESTED** [review]

File: schema/tests/Provider/Company/CompanyRepositoryTest.php [hunk 5]
```diff
@@ -21,8 +21,23 @@ public function test_runner()
         $this->it_finds_by_brandId();
         $this->it_finds_prepaid_companies();
         $this->it_finds_vpbx_company_ids_by_brand();
+        $this->it_finds_one_by_domain_users();
     }
 
+    public function it_finds_one_by_domain_users(){
```
Comment (mmadariaga) [inline schema/tests/Provider/Company/CompanyRepositoryTest.php:27]: Either call this method `public function it_finds_one_by_domain() {` or call repository method `findOneByDomainUsers`

**Review by mmadariaga — CHANGES_REQUESTED** [review]

**Review by mmadariaga — APPROVED** [review]

**Comment by ironArt3mis** [comment]
Rebase completed.

**Pull request merged**